app/Models/Device.php

```diff
@@ -387,6 +364,16 @@ public function statusName()
 
     // ---- Accessors/Mutators ----
 
+    public function getStatus()
```

Comment:
Can you please remove getStatus() and getLastPolled(), just use the property accessors: ->status and ->last_polled